src/main/webapp/WEB-INF/tags/instructor/course/courseEditAddInstructorPanel.tag

```diff
@@ -46,13 +46,12 @@
                 <div id="accessControlEditDivForInstr${addInstructorPanel.index}">
                     <div class="form-group">
                         <label class="col-sm-3 control-label">
-                            <input type="checkbox" name="<%=Const.ParamsNames.INSTRUCTOR_IS_DISPLAYED_TO_STUDENT%>" value="true" checked
-                               data-toggle="tooltip" data-placement="top" title="<%=Const.Tooltips.INSTRUCTOR_DISPLAYED_TO_STUDENT%>">
+                            <input id="displayToStudent" type="checkbox" name="<%=Const.ParamsNames.INSTRUCTOR_IS_DISPLAYED_TO_STUDENT%>" value="true" data-toggle="tooltip" data-placement="top" title="<%=Const.Tooltips.INSTRUCTOR_DISPLAYED_TO_STUDENT%>">
                             Display to students as:
                         </label>
                         <div class="col-sm-9">
-                            <input class="form-control" type="text" name="<%=Const.ParamsNames.INSTRUCTOR_DISPLAY_NAME%>"
-                                placeholder="E.g.Co-lecturer, Teaching Assistant"
+                            <input class="form-control" id="titleDisplayed" type="text" name="<%=Const.ParamsNames.INSTRUCTOR_DISPLAY_NAME%>"
```

Comment:
Use `title-displayed` as the id. In general, use hyphens for new ids and classes. Older ids and classes that violate this can be left as is.